recipes/r-puniform/meta.yaml

```diff
@@ -0,0 +1,122 @@
+{% set version = '0.2.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-puniform
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/puniform_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/puniform/puniform_{{ version }}.tar.gz
+  sha256: 92934d2c3b1eb51c74a76831d8a4789871b5aa7d38ff6878fedb436ff1c288ff
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ posix }}filesystem  # [win]
+    - {{ posix }}make
+    - {{ posix }}sed  # [win]
+    - {{ posix }}coreutils  # [win]
+    - {{ posix }}zip  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+
+  host:
+    - libblas  # [unix]
+    - liblapack  # [unix]
+    - r-base
+    - r-adgoftest >=0.3
+    - r-rcpp >=0.12.15
+    - r-rcpparmadillo
+    - r-metafor
+
+  run:
+    - blis  # [unix]
```

Comment:
```suggestion
```

Reply:
@dbast thanks for helping to figure this out!